docs/guide/relationships/getting-started.md

```diff
@@ -173,6 +173,32 @@ export default {
 }
 ```
 
+### Loading All Relationships
+
+To load all relationships, you may use the `withAll` and `withAllRecursive` methods.
+
+The `withAll` method will load all model level relationships. Note that any constraints will be applied to all top-level relationships.
+
+```js
+// Fetch models with all top-level relationships.
+store.$repo(User).withAll().get()
+
+// As above, with a constraint.
+store.$repo(User).withAll((query) => {
+  query.has('posts')
```

Comment:
I don't think we have `has` method just yet? How about we use another example that might make more sense to apply to all relationships.

```suggestion
  // This constraint will apply to all of the relationship User has. For this
  // example, all relationship will be sorted by `createdAt` field.
  query.orderBy('createdAt')
```